### DISCUSSION TITLE
Error after opening server from unbuntu/zsh  terminal(Windows)

### DISCUSSION THREAD

**Discussion by tsekeres** [nss-evening-cohort-14/evening-client-side#22] [Help Me!] [answered]
Tad Sekeres

Have you searched the other help tickets? Yes

Issue: Error message when I open my project in the server with hs -o. The window in the browser opens successfully but then quickly sends me an error in the terminal window.

Error message:
➜  week1-StarWarsofficial git:(main) ✗ hs -o
Starting up http-server, serving ./ on: http://0.0.0.0:8080
Hit CTRL-C to stop the server
[Thu, 14 Jan 2021 22:44:19 GMT] "GET /" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36"
(node:866) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
(Use `node --trace-deprecation ...` to show where the warning was created)
[Thu, 14 Jan 2021 22:44:19 GMT] "GET /main.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36"
[Thu, 14 Jan 2021 22:44:19 GMT] "GET /main.js" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36"
[Thu, 14 Jan 2021 22:44:19 GMT] "GET /favicon.ico" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36"
[Thu, 14 Jan 2021 22:44:19 GMT] "GET /favicon.ico" Error (404): "Not found"

What have you googled?

https://stackoverflow.com/questions/49724821/get-favicon-ico-error-but-can-not-find-it-in-the-code
https://stackoverflow.com/questions/18301745/how-to-set-up-a-favicon
https://www.codegrepper.com/code-examples/html/favicon.ico+not+loading+localhost

What have you tried? 
1. I tried opening in bash as well as ubuntu. Same result.
2. I tried putting a few of those code "fixes" into my head.

Copy/paste link TO THE FILE:
https://github.com/tsekeres/week1-StarWarsofficial
Please advise if this is not the right link to the file.

**Comment by tsekeres** [comment]
I saw this error in Dr T's inspect window when she looked at her starwars page in the browser during class. But it did not show up in her terminal.

**Reply by TrinityChristiana** [reply]
This is a common error while hard-refreshing that can be ignored for now. The google results you provided actually explain how to fix it if you ever wanted to come back later to try.

**Reply by tsekeres** [reply]
Related to this: When I have a browser open with my project from typing hs -o, my command line in VS Code is not usable. The cursor is there after the CTRL-C notice. But it will not run commands. I noticed in class that Dr. T's is not like this. For instance, When she switched back to her main branch to check if the background color was red/blue, she used her terminal that was open to do that.. That was to show us that both branches exist separately. Anyway. I (and others) have been opening a second terminal as a work around. And I thought I would ask since I saw that discrepancy in the video.

**Reply by drteresavasquez** [reply]
1. Does the page load?
2. I am using 2 terminals in VS code. One for my server and one for my branches. There is a plus next to the drop down. I use that to get another terminal.

If your side loads, everything is working correctly just as Trinity said. If not, give us a screen shot of your issue here.

**Reply by tsekeres** [reply]
All good then! I just didn't catch on to the second terminal! Thank you!

**Comment by drteresavasquez** [comment, marked as answer]
1. Does the page load?
2. I am using 2 terminals in VS code. One for my server and one for my branches. There is a plus next to the drop down. I use that to get another terminal.

If your site/app loads, everything is working correctly just as @TrinityChristiana  said. If not, give us a screen shot of your issue here.

**Reply by tsekeres** [reply]
All good then!

**Reply by drteresavasquez** [reply]
Wonderful! I am glad you are reading your terminal logs though! That is important, but if the server runs at this point all is GTG!
